src/drt/include/triton_route/TritonRoute.h

```diff
@@ -69,8 +69,6 @@ namespace triton_route {
 
 struct ParamStruct
 {
-  std::string guideFile;
```

Comment:
The ISPD benchmarks require us to read a guide file not generated by OR.  Is that still possible?

Reply:
Already handled, check run-ispd.py

Reply:
Nevermind I see you have a read_guides command.